ONSdigital/respondent-home-ui
# Sus020 rh headers and footers

# Motivation and Context
Update headers and footers on RH to use those in [sdc-pattern library](https://sdc-global-design-patterns.netlify.com/components/detail/header.html) to ensure user experience is consistent when passing to eQ.

# What has changed
- Replaced page header with pattern library component
- Replaced page footer with pattern library component 

Additional pattern library markup alignment:
- Updated head to use styles and images from latest version of patter library on cdn
- Replaced js scripts in RH to use those from pattern library on CDN
- Updated markup for confidentiality message to use pattern library markup
- Added PL 'skip to content' accessibility link

Bug fixes:
- Fixed broken error message anchor link

Content changes:
- Updated page content to improve UX/readability
- Pointed 'what we do' footer link to household studies 'about' content

# How to test?
- Check header and footers are correct and links work
- Cross browser test and code validation
- Check content is as agreed with social team
- Check required javascript functions still work

# Links
https://trello.com/c/UwNvY2vg/283-sus020-rh-headers-and-footers-5

# Screenshots (if appropriate):
